<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>980</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="txt_path">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gv1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>501</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>12</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PATH</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>270</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_resize_x">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_resize_y">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sbar_rotate">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_rename">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>170</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gv2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>501</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ok">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>218</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_resize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>63</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rotation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hflip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hflip</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_vflip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vflip</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_resize_ok">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_rotation_angle">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rotation_ok">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
